### ISSUE TITLE
Add support for a blurred dim view when in edit mode

### ISSUE THREAD

**Issue by Blackjacx** [Blackjacx/SHSearchBar#7]

**Comment by Blackjacx** [comment]
Consider 
• dark dim view
• blurred dim view

**Comment by narendrajagne** [comment]
Hi, Can you assign me this ?

**Comment by narendrajagne** [comment]
Thanks .. I am starting working on it.

**Comment by Blackjacx** [comment]
Thank you so much :)

**Comment by narendrajagne** [comment]
Hi can you explain me this issue in details ?

**Comment by Blackjacx** [comment]
Right now the background is darkened. My goal was when I created this ticket to also support a [blur effect ](https://www.hackingwithswift.com/example-code/uikit/how-to-add-blur-and-vibrancy-using-uivisualeffectview) additionally.

**Comment by narendrajagne** [comment]
I need work on SHSearchBarExample class or SHSearchBar package ?

**Comment by narendrajagne** [comment]
Hii, I tried working on SHSearchBar package but it is locked. How to start working on it. plz guide me

**Comment by Blackjacx** [comment]
Hi! How is it locked, what do you mean? You have to fork it before making changes.

**Comment by narendrajagne** [comment]
Yes I did , you can see on ur fork section. can we connect on discord @jagne_ios?. I need to discuss with you more.
